package.json

```diff
@@ -27,6 +27,8 @@
     "schema-drift": "babel-node --no-warnings --extensions '.ts,.js' ./scripts/schema-drift.ts",
     "start": "yarn run dev",
     "test": "yarn type-check && yarn lint && jest",
```

Comment:
Should the `test` command be updated here? I guess it doesn't have to be given that the main config pulls in the sub configs.